README.md

```diff
@@ -1,4 +1,4 @@
-# Azure Functions dapr extensions
+# DAPR Extension for Azure Functions on Azure Container Apps
```

Comment:
This extension will work on other platforms as well. I think we can remove ACA from headings. 

```suggestion
# DAPR Extension for Azure Functions
```